README.md

```diff
@@ -1 +1,204 @@
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Ruby-capstone] <a name="about-project"></a>
+
+>In this project, we will create a console app that will help to keep a record of different types of things you own: books, music albums, movies, and games. Everything will be based on the UML class diagram presented on our project. The data will be stored in JSON files but we will also prepare a database with tables structure analogical to our program's class structure.
+
+**[Ruby_project__capstone]** it shows "Catalog of my things" in a simple console app that allows users to manage collections of the things they own.
+
+## 🛠 Built With <a name="built-with-Ruby"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Ruby
+  Postgres SQL
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+-[x] Create a main.rb file that will serve as your console app entry-point.
+-[x]  Our console app, at the start, should:
+  - Present the user with a list of options to perform.
+  - Let users choose an option.
+  - If needed, ask for parameters for the option.
+  - Have a way to quit the app.
+-[x] The following options should be available:
+  - List all books
+  - List all music albums
+  - List all movies
+  - List of games
+  - List all genres (e.g 'Comedy', 'Thriller')
+  - List all labels (e.g. 'Gift', 'New')
+  - List all authors (e.g. 'Stephen King')
+  - List all sources (e.g. 'From a friend', 'Online shop')
+  - Add a book
+  - Add a music album
+  - Add a movie
+  - Add a game
+-[x] All data should be preserved by saving collections in .json files.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- VIDEO RECORDING -->
+
+## 🚀 Live Demonstration <a name="live-demo"></a>
+
+> 
+
+- [Video recording](https://drive.google.com/file/d/1B3S9A2qQulpGxnC4mW7Ozv9NcIHkThUw/view?usp=sharing)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+- [ ]  install Ruby
+- [ ]bundle init
+- [ ]bundle install
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+- [ ] Open terminal
+- [ ] git clone https://github.com/theodoku/Ruby-Capstone.git
+- [ ] cd Capstone-ruby
+  
+
+### Install
+
+Install this project with:
+
+ -[ ] Install Ruby
+ -[ ] bundle install
+ -[ ] gem install
+
+
+
+### Run tests
+
+To run tests, run the following command:
+
+-[] rspec spec
+
+
+### Deployment
```

Comment:
- [ ] Kindly include ways to run the project in the terminal. Please include the command a user would need to start the application in **_IRB terminal or shell terminal._** 👍